### PR TITLE
Sentry test - stage

### PR DESCRIPTION
**Pull request by daniel-farina**

### CODE REVIEW

File: yarn.lock
Comment (jonator) [inline yarn.lock:1]: This file should not be deleted

File: packages/web/next.config.js [hunk 1]
```diff
@@ -31,3 +31,40 @@ const withBundleAnalyzer = require("@next/bundle-analyzer")({
...
+  },
+  {
+    // For all available options, see:
+    // https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/
+
+    // Upload a larger set of source maps for prettier stack traces (increases build time)
+    widenClientFileUpload: true,
+
+    // Transpiles SDK to be compatible with IE11 (increases bundle size)
+    transpileClientSDK: true,
```
Comment (jonator) [inline packages/web/next.config.js:59]: Do we really want this?

File: packages/web/.gitignore [hunk 2]
```diff
@@ -35,3 +35,6 @@ yarn-error.log*
 
 # typescript
 *.tsbuildinfo
+
+# Sentry Auth Token
+.sentryclirc
```
Comment (jonator) [inline packages/web/.gitignore:40]: The auth token is not an ENV var?

Comment (daniel-farina) [reply packages/web/.gitignore:40]: Yes it is, already  added it. This is to ignore the sentryirc file. This PR was a test though., I told Matt to push it from his PR 👍 . Closing this PR.

**Pull request closed without merging**